apps/admin-portal/src/components/claims/add/add-dialect.tsx

```diff
@@ -0,0 +1,244 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the 'License'); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import { addDialect, addExternalClaim } from "../../../api";
+import { AddExternalClaim, AlertLevels } from "../../../models";
+import { FormValue, useTrigger } from "@wso2is/forms";
+import { Grid, Icon, Modal } from "semantic-ui-react";
+import { LinkButton, PrimaryButton, Steps } from "@wso2is/react-components";
+import React, { useState } from "react";
+
+import { addAlert } from "@wso2is/core/store";
+import { ApplicationWizardStepIcons } from "../../../configs";
+import { DialectDetails } from "../wizard/external-dialect/dialect-details-add-dialect";
+import { ExternalClaims } from "../wizard/external-dialect/external-claims-add-dialect";
+import { SummaryAddDialect } from "../wizard/external-dialect/summary-add-dialect";
+import { useDispatch } from "react-redux";
+
+/**
+ * Prop types for `AddDialect` component.
+ */
+interface AddDialectPropsInterface {
+    /**
+     * Open the modal.
+     */
+    open: boolean;
+    /**
+     * Handler to be called when the modal is closed.
+     */
+    onClose: () => void;
+    /**
+     * Function to be called to initiate an update.
+     */
+    update: () => void;
+}
+
+/**
+ * A component that lets you add a dialect.
+ * 
+ * @param {AddLocalClaimsPropsInterface} props
+ * 
+ * @return {React.ReactElement} component.
+ */
+export const AddDialect = (props: AddDialectPropsInterface): React.ReactElement => {
```

Comment:
```suggestion
export const AddDialect = (props: AddDialectPropsInterface): ReactElement => {
```

Reply:
Since I am not importing `ReactElement` separately, I need to access `ReactElement` from `React`?

Reply:
You have to import ReactElement seperately.